apps/server/src/shared/controller/transformer/sanitize-html.transformer.spec.ts

```diff
@@ -48,15 +48,25 @@ describe('SanitizeHtmlTransformer Decorator', () => {
 				'<b><mark>html <h4>text</h4></mark></b><span class="math-tex">[x=\frac{-bpmsqrt{b^2-4ac}}{2a}]</span>'
 			);
 		});
+		it('should remove attributes without values', () => {
+			const plainString = {
+				contentCk5: '<a name></a><a link></a>',
+			};
+			const instance = plainToClass(WithHtmlDto, plainString);
+			expect(instance.contentCk5).toEqual('<a></a><a></a>');
+		});
 	});
 
 	describe('when sanitizing rich text ck4 formatting', () => {
 		it('should remove all html but rich text ck4 tags', () => {
```

Comment:
I think this test name does not make sense.
I think we are checking here if all tags except the ck4 supported tags are removed.

Here script and style should be removed.

I would nice to rename this test.

Reply:
After rereading this a few times I guess I get what the text means.
But I would be much easier to use simpler language. (For us non native speakers, xD)